pythainlp/__init__.py

```diff
@@ -34,4 +34,3 @@
 )
 from pythainlp.transliterate import romanize, transliterate
 from pythainlp.util import collate, thai_strftime
-from pythainlp.benchmarks import benchmark
```

Comment:
- [ ] Add pythainlp.benchmarks

Reply:
That was my mistake, I already fixed it. 😥